### DISCUSSION TITLE
ILI9488 3,5" and 4" rotate problem

### DISCUSSION THREAD

**Discussion by eb4cuv** [Bodmer/TFT_eSPI#2716] [General]
I use a 3.5" ILI9488 TFT that works very well. The problem comes when I change it to a 4" TFT that the image is rotated 180 degrees.
I know that with tft.setrotation it is solved, but I would like to know when the application starts if the tft is rotated to program the setrotation or not.
Is there a command to check it?

thanks

**Comment by TangerineDreamer** [comment]
hi, eb4cuv,
got the same problem with some 4.0 TFT that have an initial 180° rotation,but some are not concerned with that,and i think it's before
the app begins and rotation function begins,it must be a hardware choice made by the vendor,especially for the new 4.0 versions.
